packages/blockchain/src/consensus/clique.ts

```diff
@@ -233,17 +244,24 @@ export class CliqueConsensus implements Consensus {
       }
     }
 
+    this._cliqueLatestSignerStates.sort((a, b) => (a[0] > b[0] ? 1 : -1))
+
     // save to db
     const formatted = this._cliqueLatestSignerStates.map((state) => [
       bigIntToBytes(state[0]),
       state[1].map((a) => a.toBytes()),
     ])
     await this.blockchain!.db.put(CLIQUE_SIGNERS_KEY, RLP.encode(formatted), DB_OPTS)
     // Output active signers for debugging purposes
-    let i = 0
-    for (const signer of this.cliqueActiveSigners()) {
-      debug(`Clique signer [${i}]: ${signer}`)
-      i++
+    if (signerState !== undefined) {
+      let i = 0
+      try {
+        for (const signer of this.cliqueActiveSigners(signerState[0])) {
+          debug(`Clique signer [${i}]: ${signer} (block: ${signerState[0]})`)
+          i++
+        }
+        // eslint-disable-next-line no-empty
+      } catch (e) {}
```

Comment:
What's the error that could occur here?

Reply:
I should have commented because now I don't remember. I think it was something when I tried to sync Goerli, and then for some reason `signerState[0]` was undefined, which throws the method. (If you want I can re-investigate)